zyngine/zynthian_lv2.py

```diff
@@ -293,7 +293,6 @@ def _generate_plugin_presets_cache(plugin):
 	# Get banks
 	banks = plugin.get_related(world.ns.presets.Bank)
 	for bank in banks:
-		world.load_resource(bank)
```

Comment:
This is needed because the rdfs.label is not always available without loading the resource. It depends on how the bank has been defined on TTLs.